oracle/build/dbimage/image_build.sh

```diff
@@ -239,17 +241,25 @@ execute_command() {
     CDB_NAME="${DUMMY_VALUE}"
   fi
 
+  if [ "${DB_VERSION}" == "${ORACLE_18}" ] && [ "${EDITION}" == "xe" ]; then
+    ORACLE_HOME="/opt/oracle/product/18c/dbhomeXE"
+    ORACLE_BASE="/opt/oracle"
+  else
+    ORACLE_HOME="/u01/app/oracle/product/${DB_VERSION}/db"
+    ORACLE_BASE="/u01/app/oracle"
+  fi
+
   if [ -z "${TAG}" ]; then
     TAG="gcr.io/${GCR_PROJECT_ID}/oracle-database-images/oracle-${DB_VERSION}-${EDITION}-${IMAGE_NAME_SUFFIX}:latest"
   fi
 
   if [ "${LOCAL_BUILD}" == true ]; then
-    BUILD_CMD=$(echo sudo docker build --no-cache --build-arg=DB_VERSION=${DB_VERSION} --build-arg=CREATE_CDB=${CREATE_CDB} --build-arg=CDB_NAME=${CDB_NAME} --build-arg=CHARACTER_SET=${CHARACTER_SET} --build-arg=MEM_PCT=${MEM_PCT} --build-arg=EDITION=${EDITION} --build-arg=PATCH_VERSION=${PATCH_VERSION} --tag=$TAG .)
```

Comment:
FYI, removing sudo would require users set up sudoless docker. See https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user

Reply:
Indeed!

Reply:
In my experience, most people set up sudo-less docker.